reference/docs-conceptual/community/contributing/managing-issues.md

```diff
@@ -0,0 +1,93 @@
+---
+title: How we manage issues
+description: This article explains how the PowerShell-Docs team manages pull requests.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Managing Issues
+
+This article documents how we manage issues in the PowerShell-Docs repo. This article is designed to
+be a job aid for member of the PowerShell-Docs team. It is published here to provide process
+transparency for our public contributors.
+
+## Sources of issues
+
+- Community contributors
+- Internal contributors
+- Transcriptions of comments from social media channels
+- Feedback via the Docs feedback form
+
+## Response time targets
+
+- Triaged - 5 business days
+- Fix or change - no specific target - best effort only
+
+### Labeling & Milestones
+
+#### Label Types
+
+|Prefix  | Description                                                         |
+|------- | --------------------------------------------------------------------|
+|Area    | Used to indicate what part of PowerShell or the docs that the issue is discussing.<br>Useful for feature owners to find issues for their feature.|
+|Pri     | Used to indicate the priority of the issue. Value range 0-4.        |
+|Issue   | Used to classify the type of feedback for issue                     |
+|Review  | Used for issue that require further review by the team              |
+|Status  | Used to indicate the status of the work item                        |
+|Waiting | Used to indicate that we are waiting on                             |
```

Comment:
Waiting on.... maybe add the words "external forces" or something to that effect.

Reply:
something :)